### PR TITLE
Created service__title components

### PR DESCRIPTION
**Pull request by MarcoBrunoBR**
**CHANGELOG** :memo:

Created service__title components

screen shot 2016-11-10 at 7 27 47 pm

### CODE REVIEW

File: src/scss/components/_services.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    color: get-color(punch);
+    text-align: center;
+    position: relative;
+  }
+  &__title:before,
+  &__title:after {
+    content: '';
+    display: block;
+    width: 39%;
+    border: 1px solid get-color(punch);
```
Comment (eduardojmatos) [inline src/scss/components/_services.scss]: @MarcoBrunoBR it's better to use only one side of border (`border-top` or `border-bottom`) instead `border`

File: src/scss/components/_services.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    font-size: 18px;
+    color: get-color(punch);
+    text-align: center;
+    position: relative;
+  }
+  &__title:before,
+  &__title:after {
+    content: '';
+    display: block;
+    width: 39%;
```
Comment (eduardojmatos) [inline src/scss/components/_services.scss]: @MarcoBrunoBR is ok for now, but in the future we need to think about text bigger than the current size

**Comment by GeorgeJegon** [comment]
Would be nice to had a print to see how this component looks like 😁 
For example, i don't know why we are defining `before/after` with `39%` as width value

File: src/scss/components/_services.scss (hunk 1 above)
Comment (MarcoBrunoBR) [inline src/scss/components/_services.scss]: Fair enough @eduardojmatos

**Comment by eduardojmatos** [comment]
@MarcoBrunoBR the `sass-lint` is showing some errors to fix

**Comment by eduardojmatos** [comment]
@MarcoBrunoBR :shipit:

**Pull request merged**